### PR TITLE
New style/max

### PR DESCRIPTION
**Pull request by jtlap**
all max family

### CODE REVIEW

File: include/eve/module/core/regular/absmax.hpp [hunk 1]
```diff
@@ -71,10 +92,18 @@ namespace eve
...
-#include <eve/module/core/regular/impl/absmax.hpp>
+namespace eve::detail
+{
+  template<typename T0, typename T1, typename... Ts, callable_options O>
+  EVE_FORCEINLINE constexpr common_value_t<T0, T1, Ts...>
+  absmax_(EVE_REQUIRES(cpu_), O const & o, T0 r0, T1 r1, Ts... rs) noexcept
+  {
+    return eve::abs[o.drop(pedantic2,numeric2)](eve::max[o.drop(saturated2)](r0, r1, rs...));
+  }
+}
```
[the remark this answers is not in the document]
Comment (jfalcou) [reply include/eve/module/core/regular/absmax.hpp:106]: Yeah, this was a test to see if we can win a bunch in this case. We're gonna put it back  to a file and only do that when we only have a common include in the definition file as of now.

[the remark this answers is not in the document]
Comment (DenisYaroshevskiy) [reply include/eve/module/core/regular/absmax.hpp:106]: Up to you

File: include/eve/module/core/regular/min.hpp [hunk 2]
```diff
@@ -12,6 +12,30 @@
...
+      return EVE_DISPATCH_CALL(t0,  t1, ts...);
+    }
+
+    template<kumi::non_empty_product_type Tup>
+    EVE_FORCEINLINE constexpr kumi::apply_traits_t<eve::common_value,Tup>
+    operator()(Tup const & t) const noexcept  requires(kumi::size_v<Tup> >= 2) { return EVE_DISPATCH_CALL(t); }
+
+    template<typename Callable>
+    requires(!kumi::product_type<Callable> && !eve::ordered_value<Callable>)
+    EVE_FORCEINLINE constexpr auto operator()(Callable const & f) const noexcept { return EVE_DISPATCH_CALL(f); }
```
Comment (DenisYaroshevskiy) [inline include/eve/module/core/regular/min.hpp:33]: @jfalcou do we want to change the return type here from auto, as w discussed?

Comment (jfalcou) [reply include/eve/module/core/regular/min.hpp:34]: Will do in a next PR

**Pull request merged**